src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-indent;
+  margin: 30px auto;
+  padding: $padding;
+  max-width: setWidth($cell-size, $cell-indent, $days-in-week, $padding);
+
+  @each $day, $index in $week {
+    &--start-day_#{$day} :first-child {
+      margin-left: setMargin($cell-size, $cell-indent, $index);
+    }
+
+    @for $index from $month-length through $days-in-month {
+      &--month-length_#{$index} :nth-child(n+#{$index+1}) {
+        display: none;
+      }
+    }
```

Comment:
it should be on the same level with `@each $day, $index in $week`, not inside it